src/main/java/com/ibanity/apis/client/products/xs2a/models/read/SynchronizationReadQuery.java

```diff
@@ -19,6 +19,8 @@
     private String subtype;
     private String resourceId;
     private String resourceType;
+    private boolean customerOnline;
```

Comment:
You have to update  SynchronizationServiceImpl:31 in order to pass the new properties in the request.

Reply:
split/create SynchronizationCreateQuery from SynchronizationReadQuery and map readonly read/write fields

Reply:
Alright but that's a breaking change so you'll have to update the major version.

Reply:
agreed. How do we handle versioning here?